modules/vaos/app/serializers/vaos/va_appointments_serializer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module VAOS
+  class VAAppointmentsSerializer
+    include FastJsonapi::ObjectSerializer
+
+    set_id do |object|
+      Digest::MD5.hexdigest(object.start_date)
+    end
+
+    set_type :va_appointments
+
+    attributes :start_date, :clinic_id, :clinic_friendly_name, :facility_id, :community_care
+
+    attribute :vds_appointments do |object|
+      Array.wrap(object&.vds_appointments).map do |vds|
+        vds.except(:patient_id) # remove patient identifiers
```

Comment:
Would be more verbose but what about extracting a method called `remove_patient_identifiers` instead of adding comments to describe what's happening?

Reply:
opted to keep this one the same for now too, it reads fine. If there was more going on would definitely extract into methods.